lib/backend/pgbk/common/utils.go

```diff
@@ -262,6 +292,7 @@ func SetupAndMigrate(
 		log.WithError(err).Debugf("Failed to confirm the existence of the %v table.", tableName)
 	}
 
+	var schemas []string
```

Comment:
Does this need to live outside of the closure in RetryTx?

Reply:
Yes, after exiting the RetryTx we log the original and final version. I could only expose the lenght but it would take a new variable and I'm was not sure how relevant this was.